src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -24,7 +24,7 @@
 
 import hou
 
-IGNORE_REF_VALUES = ("opdef:", "oplib:", "temp:")
+IGNORE_REF_VALUES = ("opdef:", "oplib:", "temp:", "op:")
```

Comment:
Did you do any additional digging or find more references that `op:` always points to another node and never somehow evaluates to a real path? Just want to make sure we aren't missing anything by adding this to the skip list.

Reply:
I found [this page](https://www.sidefx.com/docs/houdini/io/op_syntax.html) which sounds like `op:` references are always internal scene references and not files. I'll double check with an SME.

Reply:
Confirmed offline that we should be covered here and find to skip `op:`